Makefile

```diff
@@ -19,7 +19,11 @@ build:
 
 .PHONY: migrate-db
 migrate-db:
-	deno run --allow-net --allow-read --allow-env migrate-db.ts
+	deno task migrate-db
+
+.PHONY: run
+run:
+	deno task run
```

Comment:
I'd suggest removing this, I like to keep `Makefile` to a few, important (and more common) commands.